src/ManageCourses.Api/Services/EnrichmentService.cs

```diff
@@ -20,68 +20,23 @@ public EnrichmentService(IManageCoursesDbContext context)
-        /// <param name="publishableOnly">if there is a draft enrichment then get the ucas contact details</param>
-        /// <returns>
-        /// An enrichment object with the enrichment data.
-        /// If there is nothing in the db then an empty record with Ucas contact details will be returned.
-        /// </returns>
-        public UcasInstitutionEnrichmentGetModel GetInstitutionEnrichment(string instCode, string email, bool publishableOnly)
+        public UcasInstitutionEnrichmentGetModel GetInstitutionEnrichment(string instCode, string email)
         {
-            ValidateUserOrg(email, instCode);
-
-            instCode = instCode.ToUpperInvariant();
-
-            var enrichmentsQuery = _context.InstitutionEnrichments
-                .Where(ie => ie.InstCode == instCode);
-
-            if (publishableOnly)
-            {
-                enrichmentsQuery = enrichmentsQuery.Where(ie => ie.Status == EnumStatus.Published);
-            }
-
-            var enrichment = enrichmentsQuery.OrderByDescending(x => x.Id)
-                .Include(e => e.CreatedByUser)
-                .Include(e => e.UpdatedByUser)
-                .FirstOrDefault();
-
-            var enrichmentGetModel = _converter.Convert(enrichment) ?? new UcasInstitutionEnrichmentGetModel();
-
-            var enrichmentModel = enrichmentGetModel.EnrichmentModel;
-
-            var useUcasContact =
-                string.IsNullOrWhiteSpace(enrichmentModel.Email) &&
-                string.IsNullOrWhiteSpace(enrichmentModel.Telephone) &&
-                string.IsNullOrWhiteSpace(enrichmentModel.Website) &&
-                string.IsNullOrWhiteSpace(enrichmentModel.Address1) &&
-                string.IsNullOrWhiteSpace(enrichmentModel.Address2) &&
-                string.IsNullOrWhiteSpace(enrichmentModel.Address4) &&
-                string.IsNullOrWhiteSpace(enrichmentModel.Address3) &&
-                string.IsNullOrWhiteSpace(enrichmentModel.Postcode);
-
-            if (useUcasContact)
-            {
-                var ucasInst = _context.UcasInstitutions.SingleOrDefault(x => x.InstCode == instCode);
-                if (ucasInst != null)
-                {
-                    enrichmentModel.Email = ucasInst.Email;
-                    enrichmentModel.Telephone = ucasInst.Telephone;
-                    enrichmentModel.Website = ucasInst.Url;
-                    enrichmentModel.Address1 = ucasInst.Addr1;
-                    enrichmentModel.Address2 = ucasInst.Addr2;
-                    enrichmentModel.Address3 = ucasInst.Addr3;
-                    enrichmentModel.Address4 = ucasInst.Addr4;
-                    enrichmentModel.Postcode = ucasInst.Postcode;
-                }
-            }
-
-            return enrichmentGetModel;
+            return GetInstitutionEnrichment(instCode, email, false);
         }
+        public UcasInstitutionEnrichmentGetModel GetInstitutionEnrichmentForPublish(string instCode, string email)
```

Comment:
Could of done that

public UcasInstitutionEnrichmentGetModel GetInstitutionEnrichmentForPublish(string instCode, string email) =>
            return GetInstitutionEnrichment(instCode, email, true);